app/src/views/GrantRound.vue

```diff
@@ -0,0 +1,305 @@
+<template>
+  <!-- GrantRound details -->
+  <div v-if="!isContributing">
+    <div v-if="grantRound.address">
+      <h1 class="my-6 text-center text-3xl font-extrabold text-gray-900">
+        Details for Grant Round: <span :title="grantRound.address">{{ formatAddress(grantRound.address) }}</span>
+      </h1>
+      <p>Name: {{ grantRoundMetadata?.name }}</p>
+      <p>Description: {{ grantRoundMetadata?.description }}</p>
+      <p>Status: {{ grantRound.status }}</p>
+      <p>Matching: {{ grantRoundMetadata?.matchingAlgorithm }}</p>
+      <p>Website: {{ grantRoundMetadata?.properties?.websiteURI }}</p>
+      <p>Governance: {{ grantRoundMetadata?.properties?.governanceURI }}</p>
+      <p>Twitter: {{ grantRoundMetadata?.properties?.twitterURI }}</p>
+      <p>sponsors: {{ grantRoundMetadata?.properties?.sponsorsURI }}</p>
+      <p>Img: {{ grantRoundMetadata?.logoURI }}</p>
+
+      <p>
+        Funds:
+        <a
+          class="link"
+          :title="grantRound.matchingToken.name"
+          :href="`https://etherscan.io/token/${grantRound.matchingToken.address}?a=${grantRound.address}`"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          {{ grantRound.funds.toString() }} {{ grantRound.matchingToken.symbol }}
+        </a>
+      </p>
+      <p>
+        {{ hasStatus('Upcoming')(grantRound) ? 'Will begin' : 'Started' }}:
```

Comment:
Feels clunky that this is two function calls, instead of one function call with a single argument. @gdixon I thought we had changed this a few weeks ago? I think it'd be more clear if this was `hasStatus('Upcoming', grantRound)`

Reply:
I was using currying here because it neatens up the sorts, but if its being used for other things then we can make that change!

Reply:
Nope, fine by me! Just misremembered what we said when I saw this last time 🙂